Add JustMockLite to additional resources (#19284)
dotnet/AspNetCore.Docs: aspnetcore/mvc/controllers/testing.md

monikerRange: '>= aspnetcore-2.1'
ms.author: riande
ms.custom: mvc

> ms.date: 22/07/2020

Reviewer: Change the date format to `7/22/2020` ...

```suggestion
ms.date: 7/22/2020
```

* <xref:test/integration-tests>
* [Create and run unit tests with Visual Studio](/visualstudio/test/unit-test-your-code)
* [MyTested.AspNetCore.Mvc - Fluent Testing Library for ASP.NET Core MVC](https://github.com/ivaylokenov/MyTested.AspNetCore.Mvc): Strongly-typed unit testing library, providing a fluent interface for testing MVC and web API apps. (*Not maintained or supported by Microsoft.*)

> * [JustMockLite](https://github.com/telerik/JustMockLite): The most powerful free mocking framework available for .NET developers. (*Not maintained or supported by Microsoft.*)

Reviewer: Remove the marketing language ...

```suggestion
* [JustMockLite](https://github.com/telerik/JustMockLite): A free mocking framework available for .NET developers. (*Not maintained or supported by Microsoft.*)
```
